proto/flexstat.proto

```diff
@@ -0,0 +1,119 @@
+syntax = "proto3";
+package xbospb;
+import "nullabletypes.proto";
+
+message FlexstatActuationMessage {
+	uint64 time = 1;
+	repeated FlexstatSetpoints setpoints = 2;
```

Comment:
Just curious: why are there multiple setpoints in a message with a given timestamp?

Reply:
So: our MPC controller publishes a list of setpoints for the next 24 hours. I haven't pushed the updated thermostat driver that does the actual controls yet, but that driver will handle this.

Reply:
In that case, the setpoint message should have a field that indicates the time (+ duration) for the setpoint. I think there is a "setpoint schedule" message already defined in the `iot.proto` file

Reply:
There is a message called change_time in FlexstatSetpoints that indicates the time of change for each setpoint (line #11). duration might be a good idea. Let me check with the MPC team. Thanks!

Oh! I'll take a look at the setpoint schedule though.